conda-forge.yml

```diff
@@ -4,3 +4,8 @@ github:
 conda_forge_output_validation: true
 conda_build:
   pkg_format: '2'
+
+build_platform:
+  osx_arm64: osx_64
```

Comment:
If you want to add osx-arm, you'll have to have more changes (e.g. host deps need to be available in build env). I would suggest to do that in a separate PR.